test/subspace-test-service/src/lib.rs

```diff
@@ -766,7 +766,7 @@ impl MockConsensusNode {
 
     /// Wait for the operator finish processing the consensus block before return
     pub async fn confirm_block_import_processed(&mut self) {
-        // Send one more notification to ensure the previous consensus block import notificaion
+        // Send one more notification to ensure the previous consensus block import notification
         // have received by the operator
```

Comment:
I think this should be "notifications were received" or "notifications have been received"?

Reply:
```suggestion
        // have been received by the operator
```